### ISSUE TITLE
More "how to use it" examples ? 

### ISSUE THREAD

**Issue by JulienDesmettre** [wadackel/react-stack-grid#50]
Hello, thanks a lot for these components. Your live demo is really nice.
It's not an issue, more an understanding problem by my side, as i'm a newbie with React, but seeing more complete component code and how you use different properties and how you control your grid will be really helpfull. Or at least seeing how your live demo is made, especially the "real world" example.  I'm trying to understand it through react developper tool, but i'm not recognizing the code structure.

**Comment by nick-gaudreau** [comment]
I agree. Could you share the react code for the "real world" example

**Comment by NevenLeung** [comment]
Hi @JulienDesmettre @JulienDesmettre , here is the code of [real world example](https://github.com/tsuyoshiwada/react-stack-grid/blob/gh-pages/js/pages/RealWorld.js).	

Some props may be not necessary. Here is the my minimal usage:

```
<StackGrid
  columnWidth={210}
  monitorImagesLoaded={true}
>
  {PhotoItems}  // PhotoItems is an array of <img> component
</StackGrid>
```

If you have any other requirement, check the usage of the props properties in README.md.
